thewalrus/labudde.py

```diff
@@ -12,8 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-This module implements the La Budde's algorithm to calculate the
+La Budde's algorithm
+====================
+
+**Module name:** :mod:`thewalrus.labudde`
+
+.. currentmodule:: thewalrus.labudde
```

Comment:
This module breaks the implicit naming convention of the walrus, in which modules are named after the computation they _do_, rather than the name of the algorithm. Going by that covention, we should rename this module to `thewalrus.charpoly` or something.

Note that this is also the convention NumPy uses (naming functions and modules based on the problem they are solving or the quantity they return), which makes it easier for users to find what they are looking for.

Reply:
That makes sense. I'll rename it.